lib/travis/hub/serialize/keen/job.rb

```diff
@@ -0,0 +1,92 @@
+# This is what Gatekeeper sends:
+#
+#   {
+#     requests: [
+#       {
+#         event_type: 'push',
+#         matrix_size: 2,
+#         repository_id: 123,
+#         owner_id: 234,
+#         owner_type: 'User',
+#         owner: ['User', 234],
+#         language: 'default',
+#         language_version: {
+#           ruby: ['2.1.2', '2.2.2']
+#         github_language: nil,
+#         uses_sudo: false,
+#         uses_apt_get: false,
+#         dist_name: 'default',
+#         group_name: 'default'
+#       }
+#     ],
+#     deployments: [
+#       {
+#         provider: 's3',
+#         repository_id: 123
+#       }
+#     ],
+#     notifications: []
+#   }
+require 'date'
+
+module Travis
+  module Hub
+    module Serialize
+      module Keen
+        class Job < Struct.new(:job)
+          def data
+            {
+              jobs: [
+                {
+                  id:                 job.id,
+                  repository_id:      repo.id,
```

Comment:
could we also add:
`owner_type`
`owner_id`
`owner_name`
`repository_name`

I think we can also remove the `repository_slug` field.